Storage/DoctrineORMStorage.php

```diff
@@ -25,7 +26,7 @@ public function translationsTablesExist()
         $connection = $em->getConnection();
 
         // listDatabases() is not available for SQLite
-        if ('pdo_sqlite' !== $connection->getDriver()->getName()) {
+        if (!$connection->getDriver() instanceof SQLiteDriver::class) {
```

Comment:
`::class` is not required for `instanceof`

Reply:
sorry, was too quick